UnitySDK/Assets/TiltBrush/Scripts/Editor/ModelImportSettings.cs

```diff
@@ -104,11 +104,20 @@ public class ModelImportSettings : AssetPostprocessor {
 
     if (m_Info.tiltBrushVersion >= new Version { major=10 }) {
       var importer = assetImporter as ModelImporter;
+#if UNITY_2019_1_OR_NEWER
+      if (importer != null && importer.optimizeMeshPolygons && importer.optimizeMeshVertices) {
```

Comment:
We need to disable optimizeMeshVertices always, because shaders use SV_VertexId. As written, the code won't disable optimizeMeshVertices if the mesh happens to have optimizeMeshPolygons=false. It should either be
```
  // force both off
  if (importer && (optimizePoly || optimizeVert)) {
    optimizePoly = optimizeVert = fase;
  }
```

or (preferably)

```
  // force optimizeVert off
  if (importer && optimizeVert) {
    optimizeVert = false;
  }
```